### ISSUE TITLE
Mod Causing Regular Enemies & Objects To Randomly Drop Items

### ISSUE THREAD

**Issue by vizthex123** [TotallySheen/ROR2Mods#1]
Title. 

The exploding towers in Rallypoint Delta keep dropping Little Disciple, and enemies almost always drop Silence Between Two Strikes (even if they aren't elites)

Removing or disabling the mod fixes it.

**Comment by NPCPotPie** [comment]
Also having this issue. Normal elite monsters are dropping their aspects at largely increased rates as if they count as bosses. (Which is cool, but appears completely unintended by this mod's description)

**Comment by vizthex123** [comment]
Yeah, there's no fix until the developed releases an update sadly.

Just disable/uninstall it for now.

**Comment by NPCPotPie** [comment]
I already have and understand that it won't be fixed until they update with a hotfix. That's the reason for this ticket: to let the developer know that users are having an issue and how it's affecting us/them.

**Comment by vizthex123** [comment]
Ah ok, makes sense. I do hope it gets fixed, this mod seems perfect
